code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -68,9 +63,9 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 	///Minimum pressure allower for release_pressure var
 	var/can_min_release_pressure = (ONE_ATMOSPHERE * 0.1)
 	///Max amount of heat allowed inside of the canister before it starts to melt (different tiers have different limits)
-	var/heat_limit = 5000
+	var/heat_limit = 50000
 	///Max amount of pressure allowed inside of the canister before it starts to break (different tiers have different limits)
-	var/pressure_limit = 46000
+	var/pressure_limit = 500000
```

Comment:
this may need balancing

Reply:
from my tests you'd reach this amount easily by just burning o2 and plasma, do you want me to lower/increase it?